_data/sites.yml

```diff
@@ -2192,6 +2192,11 @@
   url: https://xwx.moe/
   size: 414
   last_checked: 2022-05-15
+  
+- domain: yalinpala.dev
+  url: https://yalinpala.dev/
+  size: 395
+  last_checked: 2022-05-16 (YYYY-MM-DD)
```

Comment:
I'm not sure if this line breaks the parser, but the dateformat shouldn't be here.🙂